index.html

```diff
@@ -0,0 +1,206 @@
+          </div>
+          <p class='Instructor-info'>Lorem ipsum dolor sit amet consectetur adipisicing elit. Iure maiores minima eveniet sapiente. Iusto nobis fugit dolores? Iure soluta minima laudantium maiores dolorem nobis omnis itaque eius, rerum, eligendi aut?
+          Sit ab facilis, praesentium </p>
+        </div>
+      </div>
+
+      <div class="instructor-card5" id='instructor-card5'>
+        <div class='photo-container'>
+          <a href=""><img src="images/milla.jpg" alt="instructor photo" class='instructor-photo'></a>
+        </div>
+        <div class='instructor-profile'>
+          <div class='name-job'>
+            <h4 class='instructor-name'>Million Ayalew</h4>
+            <span class='job'>Graduate of Software Engineering from HU</span>
+          </div>
+          <p class='Instructor-info'>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Ad deserunt ducimus maiores aperiam, odit veniam rerum laudantium placeat facilis cum asperiores odio id pariatur nesciunt ipsa aliquam laboriosam nostrum magnam?</p>
+        </div>
+      </div>
+
+      <div class="instructor-card6" id='instructor-card6'>
+        <div class='photo-container'>
+          <a href=""><img src="images/Addis.jpg" alt="instructor photo" class='instructor-photo' id='instructor-photo'></a>
+        </div>
+        <div class='instructor-profile'>
+          <div class='name-job'>
+            <h4 class='instructor-name' id='instructor-name'>Bedada Bekele</h4>
+            <span class='job' id='job'>Graduate of Software Engineering from AASTU</span>
+          </div>
+          <p class='Instructor-info' id='Instructor-info'>Lorem, ipsum dolor sit amet consectetur adipisicing elit. Eos sapiente illo tenetur, quo deserunt fuga? Omnis numquam illum laborum culpa nemo nesciunt quis distinctio unde ullam dolorem. Voluptate, laudantium officiis.
+          </p>
+        </div>
+      </div> 
+      <div class='more-btn'>
+      <button type="button" class="btn" id="btn">MORE <span> <i class="fa fa-angle-down"></i></span></button>
+      </div>
+ </div>
+ </section>
+  </main>
+  <footer>
+  <div class='partner-section'>
+    <h3 class='footer-title'>Partner</h3>
+    <h4 class='indicator-footer'></h4>
+    <ul class='partners-list'>
+      <li class="partner"><a href="" class='mozila'>Mozila</a></li>
+      <li class="partner"><a href="" class='google'>Google</a> </li>
+      <li class="partner"><a href="" class='naver'>Naver</a></li>
+      <li class="partner"><a href="" class='daumk'>Daumkakao</a></li>
+      <li class="partner"><a href="" class='daumk'><i class="fa-brands fa-airbnb"></i> Airbnb</a></li>
+    </ul>
+  </div>
+  <div class='foot-note'>
+    <h4 class='foot-title'>Web Bootcamp</h4>
+    <div class='address'>
+    <p class='English-version'>@2022 PCEHR Bootcamp</p>
+    <p class='korea-version'>Get Registered now</p>
+    </div>
+  </div>
+  </footer>
```

Comment:
Great job 👏🏽
- [ ] Unfortunately, the design accomplished doesn't match the one shared in the [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) project guidelines. see below the expected footer design for both the Home page 
**Home**
<img width="987" alt="Screenshot 2022-07-29 at 03 27 21" src="https://user-images.githubusercontent.com/31113416/181659271-03be967a-79e7-4b86-b916-b6edbf43affd.png">